main.nf

```diff
@@ -68,8 +68,8 @@ if( !(workflow.runName ==~ /[a-z]+_[a-z]+/) ){
 ch_output_docs = Channel.fromPath("$baseDir/docs/output.md")
 
 // Validate inputs
-params.spectra = params.spectra ?: { log.error "No read data privided. Make sure you have used the '--spectra' option."; exit 1 }()
-params.database = params.database ?: { log.error "No read data privided. Make sure you have used the '--database' option."; exit 1 }()
+params.spectra = params.spectra ?: { log.error "No spectra data provided. Make sure you have used the '--spectra' option."; exit 1 }()
```

Comment:
Oops :)
Good catch.

Reply:
hehe yes, I didnt see it in the review either.

Reply:
Funfact: This is also present in MHCquant haha